app/developers/forms.py

```diff
@@ -8,7 +8,7 @@
 
 
 class PreviewCollectionForm(FlaskForm):
-    submit = SubmitField("Preview this collection", widget=GovSubmitInput())
+    submit = SubmitField("Preview a submission", widget=GovSubmitInput())
```

Comment:
I think this continues to make sense in the context it was in and doesn't need changing - you're configuring a "collection" by setting up sections/ forms/ questions etc. and this button is allowing you to preview how that collection all comes together into a task list

Reply:
I guess _technically_ it should previously have said "Preview this schema"

Reply:
The fact that it creates a submission in order to allow you to preview the behaviour feels a bit like an underlying detail

Reply:
ok ok fair, will revert 👍